astropy/nddata/utils.py

```diff
@@ -633,11 +633,11 @@ def __init__(self, data, position, shape=None, side_length=None, wcs=None,
         self.input_position_original = position
         self.shape_input = shape
 
```

Comment:
@adonath - Without digging too deep, I assume from [#3823 (comment)](https://github.com/astropy/astropy/pull/3823#issuecomment-123929382) @larrybradley's change was just made before it was decided to go with the Numpy indexing order `(z, y, x)` rather than `(x, y, z)` in #3909. So the formatting in `Cutout2D` was consistent as `((xmin, xmax), (ymin, ymax))`, but then we changed the ordering to `((ymin, ymax), (xmin, xmax))`.